netmiko/ssh_autodetect.py

```diff
@@ -347,7 +347,7 @@ def _send_command_wrapper(self, cmd):
             return cached_results
 
     def _autodetect_remote_version(
-        self, search_patterns=None, re_flags=re.IGNORECASE, priority=99
+        self, search_patterns=None, re_flags=re.IGNORECASE, priority=99, *args, **kwargs
```

Comment:
This would need to be just **kwargs (i.e. you can't pass in *args after the named arguments search_patterns=None, re_flags=X, and priority=Y).

Reply:
fixed